xml/System.Net.Http/HttpClient.xml

```diff
@@ -112,37 +112,14 @@ On each platform, <xref:System.Net.Http.HttpClient> tries to use the best availa
 Users can also configure a specific transport for <xref:System.Net.Http.HttpClient> by invoking the <xref:System.Net.Http.HttpClient.%23ctor*> constructor that takes an <xref:System.Net.Http.HttpMessageHandler>.
 
 ### HttpClient and .NET Core
-
 Starting with .NET Core 2.1, the <xref:System.Net.Http.SocketsHttpHandler?displayProperty=nameWithType> class instead of `HttpClientHandler` provides the implementation used by higher-level HTTP networking classes such as `HttpClient`. The use of <xref:System.Net.Http.SocketsHttpHandler> offers a number of advantages:
-
 - A significant performance improvement when compared with the previous implementation.
-
 - The elimination of platform dependencies, which simplifies deployment and servicing. For example, `libcurl` is no longer a dependency on .NET Core for macOS and .NET Core for Linux.
-
 - Consistent behavior across all .NET platforms.
 
-If this change is undesirable, you can configure your application to use the older <xref:System.Net.Http.HttpClientHandler?displayProperty=nameWithType> instead in a number of ways:
-
-- By calling the <xref:System.AppContext.SetSwitch%2A?displayProperty=nameWithType> method as follows:
+### Configure behavior using run-time configuration options
 
-   ```csharp
-   AppContext.SetSwitch("System.Net.Http.UseSocketsHttpHandler", false);
-   ```
-
-   ```vb
-   AppContext.SetSwitch("System.Net.Http.UseSocketsHttpHandler", False)
-   ``` 
-- By defining the `System.Net.Http.UseSocketsHttpHandler` switch in the *.netcore.runtimeconfig.json* configuration file:
-
-  ```json
-  "runtimeOptions": {
-    "configProperties": {
-        "System.Net.Http.UseSocketsHttpHandler": false
-    }
-  }
-  ```
-  
-- By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_USESOCKETSHTTPHANDLER` and setting it to either `false` or 0.
+Certain aspects of <xref:System.Net.Http.HttpClient>'s behavior are customizable through [Run-time configuration options](/dotnet/core/run-time-config/networking). However, the behavior of these switches differs through .NET versions. For example, in .NET Core 2.1 - 3.1, you can configure whether <xref:System.Net.Http.SocketsHttpHandler> is used by default, but that option is no longer available starting in .NET 5.0.
```

Comment:
Should the `UseSocketsHttpHandler` switch be also marked as available only till .NET Core 3.1 in the referenced docs https://docs.microsoft.com/en-us/dotnet/core/run-time-config/networking?

And would it make sense to mention that using `WinHttpHandler` is still possible via OOB nuget package (manually setting it to `HttpClient` ctor)?

Reply:
> Should the `UseSocketsHttpHandler` switch be also marked as available only till .NET Core 3.1 in the referenced docs https://docs.microsoft.com/en-us/dotnet/core/run-time-config/networking?

I'm not a big fan of documenting the same thing in redundant ways, so in this PR I'm replacing the detailed docs of `UseSocketsHttpHandler` in  `HttpClient`'s remarks section by a reference to the runtime switches page, which is being updated in dotnet/docs#21197. Does this answer your question, or did you mean something different?


> And would it make sense to mention that using `WinHttpHandler` is still possible via OOB nuget package (manually setting it to `HttpClient` ctor)?

Addressed in https://github.com/dotnet/dotnet-api-docs/pull/5028/commits/e2285e6c11ca8b073480dbbac69228d75037e19d

Reply:
No, that's perfect! I wasn't aware of the other PR and the existing note. Thank you!